### PR TITLE
PPC-AD to PPC2-AD

### PR DESCRIPTION
**Pull request by ailZhou**
### Description

PPC-AD is now PPC2-AD and there are some content & rate label changes.

### Related ticket(s)

CMDCT-3395

---
### How to test

1. Sign into QMR
2. Create a child coreset
3. There should be no PPC-CH and it should be PPC2-AD in the measure list.
4. Select PPC2-AD
5. In Measure Specification section, select the first bullet to open up Performance Section
6. Refer to ticket for content changes: https://jiraent.cms.gov/browse/CMDCT-3395

### Important updates

N/A

---
### Author checklist

- [x] I have performed a self-review of my code
- [x] I have added [thorough](https://shorturl.at/aejkF) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

**Review by britt-mo — APPROVED** [review]
Confirmed locally that changes match the card AC

**Review by keeysnc — APPROVED** [review]
Few suggestions but everything looks good to me!

File: tests/cypress/cypress/e2e/a11y/ppcadPage.spec.ts [hunk 1]
```diff
@@ -1,9 +1,7 @@
-import { testingYear } from "../../../support/constants";
-
 describe("PPC-AD Page 508 Compliance Test", () => {
```
[the remark this answers is not in the document]
Comment (ailZhou) [reply tests/cypress/cypress/e2e/a11y/ppcadPage.spec.ts:1]: this is a copy of PPC-AD test saved to be tested for 2023 ppc-ad, so the text here is correct.

File: tests/cypress/cypress/e2e/a11y/ppcadPage.spec.ts [hunk 2]
```diff
@@ -1,9 +1,7 @@
...
   it("Check a11y on PPCAD Page", () => {
```
[the remark this answers is not in the document]
Comment (ailZhou) [reply tests/cypress/cypress/e2e/a11y/ppcadPage.spec.ts:2]: this is a copy of PPC-AD test saved to be tested for 2023 ppc-ad, so the text here is correct.

**Review by benmartin-coforma — APPROVED** [review]
You covered everything I could think of, plus some things I would have missed. Looks great.

**Pull request merged**